<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changePassword</class>
 <widget class="QWidget" name="changePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>修改密码</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>491</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="usernameLabel">
      <property name="font">
       <font>
        <family>AaJHNDMCC (Non-Commercial Use)</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="oldPasswordLabel">
      <property name="font">
       <font>
        <family>AaJHNDMCC (Non-Commercial Use)</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>旧密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="newPasswordLabel">
      <property name="font">
       <font>
        <family>AaJHNDMCC (Non-Commercial Use)</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>新密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="newPasswordAgainLabel">
      <property name="font">
       <font>
        <family>AaJHNDMCC (Non-Commercial Use)</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>再次输入新密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="usernameInput"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="oldPasswordInput">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="newPasswordInput">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="newPasswordAgainInput">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>630</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AaJHNDMCC (Non-Commercial Use)</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
